ja2003.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Jahresarbeit 2003
+Comment=Find die größte Fläche von zusammenhängenden Quadraten
+Categories=Game;LogicGame;
+Icon=com.github.christianrauch.Jahresarbeit-2003
+Exec=/app/ja2003
```

Comment:
`Exec` is never an absolute path.

Reply:
This path has to be absolut because the executable is not in the PATH and is has to be in the root of the folder to discover the resources.

Reply:
this comment still stand and there are ways to do it properly with a wrapper script.

Reply:
I still do not see the advantage of adding yet another file just for changing the path. I will not be able to reuse the *.desktop file anyway because of the clunky icon name `com.github.christianrauch.Jahresarbeit-2003`. The app will work perfectly fine and for a user it does not matter where the file is inside the flatpak.

Am I missing something here? Is there some drawback when starting the executable from the root path? AFAIK, a lot of Steam games work this way and there is no problem with this.

Reply:
I tried to change the `PATH` using:
```yaml
finish-args:
- --env=PATH=/app
```
and this seems to work for the *.desktop launcher, but it will not be recognised by `command: /app/ja2003`. If there is a way to set `PATH` such that it will work in the launcher and the manifest, I am happy to do so.

Reply:
I'm going to mark it as resolved, because other than presumably bad style, it's actually going to work.